src/client/components/Inputs/Select/hooks/useComponents.tsx

```diff
@@ -35,13 +35,13 @@ export const useComponents = (props: SelectProps): Returned => {
     }
     if (isMulti && selectableGroups) components.GroupHeading = SelectableGroupHeading
     if (collapsibleGroups) components.Group = CollapsibleGroup
-    if (Objects.isFunction(onPaste) || !Objects.isEmpty(inputHidden)) {
+    if (!Objects.isEmpty(inputHidden)) {
       components.Input = (originalInputProps: InputProps) => {
         // eslint-disable-next-line react/jsx-props-no-spreading
-        return <originalComponents.Input {...originalInputProps} isHidden={inputHidden} onPaste={onPaste} />
```

Comment:
Having the onPaste function here caused bugs because if it changes, the Input component is replaced with a new instance and thus loses its state. That caused problems with the default behavior, like the cursor jumping or the select list not hidden onBlur.

react-select doesn't support another way to pass the onPaste event. The other option would have been to extend the props of the native select and pass the onPaste there, but that seemed too hacky. Instead I removed it and added the onPaste handler to the Taxon div container.